helm-charts/common/vllm/templates/configmap.yaml

```diff
@@ -21,6 +21,7 @@ data:
   NUMBA_CACHE_DIR: "/tmp"
   HF_HOME: "/tmp/.cache/huggingface"
   XDG_CONFIG_HOME: "/tmp"
+  TORCHINDUCTOR_CACHE_DIR: "/tmp/pytorchinductor_cache"
```

Comment:
You could add here a comment such as "PyTorch requires root user if cache files are not owned by user".